armi/nuclearDataIO/cccc.py

```diff
@@ -671,3 +671,23 @@ def writeAscii(cls, lib, fileName):
     @classmethod
     def _write(cls, lib, fileName, fileMode):
         raise NotImplementedError()
+
+
+def getBlockBandwidth(m, nintj, nblok):
+    """
+    Return block bandwidth JL, JU from CCCC interface files.
```

Comment:
Im not sure i understand what "bandwidth" means... is it like a stride of some sort?

Reply:
yeah... I'll clarify